third_party/openmetadata/install_OM.sh

```diff
@@ -1,39 +1,82 @@
 #!/bin/bash
 
+OPERATION=install
 OM_VERSION=0.12.1
+K8S_TYPE=kind
 export OPENMETADATA_AIRFLOW_VERSION=0.12.1
 export OPENMETADATA_HELM_CHART_VERSION=0.0.39
 
 export FYBRIK_BRANCH="${FYBRIK_BRANCH:-master}"
 export FYBRIK_GITHUB_ORGANIZATION="${FYBRIK_GITHUB_ORGANIZATION:-fybrik}"
 
-if [ $# -gt 1 ]; then
-    echo "Usage: "$0" [open-metadata-version]"
-    exit -1
+if [ $# -gt 3 ]; then
+    echo "Usage: . ./install_OM.sh [install/getFiles/cleanup] [ibm-openshift/kind] [open-metadata-version]"
+    return
 fi
 
-if [ $# -eq 1 ]; then
-    OM_VERSION=$1
+if [ $# -ge 1 ]; then
+    OPERATION=$1
+fi
+
+if [ $# -ge 2 ]; then
+    K8S_TYPE=$2
+fi
+
+if [ $# -ge 3 ]; then
+    OM_VERSION=$3
+fi
+
+supported_operations=(install getFiles cleanup)
+if [[ ! ${supported_operations[*]} =~ ${OPERATION} ]]; then
+    echo supported operations: ${supported_operations[*]}
+    echo ${OPERATION} not supported
+    return
+fi
+
+if [ $OPERATION == "cleanup" ]; then
+    if [ -z ${TMP_OM_INSTALLATION_DIR+x} ]; then
+        echo could not find temporary installation dir. exiting
+        return
+    fi
+    rm -Rf ${TMP_OM_INSTALLATION_DIR}
+    echo removed directory ${TMP_OM_INSTALLATION_DIR}
+    return
 fi
 
 supported_om_version=(0.12.1)
 if [[ ! ${supported_om_version[*]} =~ ${OM_VERSION} ]]; then
     echo supported OM versions are: ${supported_om_version[*]}
-    exit -1
+    return
 fi
 
 # create temp directory
 tmp_dir=$(mktemp -d)
+echo about to download installation files to $tmp_dir
 
 # download files to temp directory
-cd $tmp_dir
-
-files_to_download=(Makefile Makefile.env pv1.yaml pv2.yaml values-deps.yaml)
+if [ $K8S_TYPE == "ibm-openshift" ]; then
```

Comment:
it's a cool tick, but you cannot run the installation script locally, at least for openshift.